spec/routing/curation_concerns_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+RSpec.describe CurationConcerns::GenericWorksController do
+  it "routes to manifest" do
+    expect(get: '/concern/generic_works/1234/manifest')
+      .to route_to(controller: 'curation_concerns/generic_works', action: 'manifest', id: '1234')
```

Comment:
This is contrary to rubocop defaults. https://github.com/bbatsov/rubocop/blob/master/config/default.yml#L384 @cbeer what should we do?